src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/CuratedMainPagePageObject.java

```diff
@@ -38,15 +36,21 @@
 
     private int value;
 
-    private Settings(int value) {
+    Settings(int value) {
       this.value = value;
     }
   }
 
-  public CuratedPageObject(WebDriver driver) {
+  public CuratedMainPagePageObject(WebDriver driver) {
     super(driver);
   }
 
+  public int getElementHeight(String element) {
+    JavascriptExecutor js = (JavascriptExecutor) driver;
+    return Integer
+        .parseInt(js.executeScript("return $('" + element + "').offset().top").toString());
```

Comment:
check if $element.height() isn't enough? Isn't casting to Integer enough? (Integer) js.executeScript...